packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -56,7 +56,7 @@ class EmbeddedContentConsumer extends React.Component {
 
   addEventListener(eventName, eventHandler) {
     if (eventName && eventHandler) {
-      this.xfcFrame.on(eventName, eventHandler);
+      this.xfcFrame.on(eventName, () => eventHandler(this.xfcFrame));
```

Comment:
Fix bug to pass the frame such that consumers can send custom events to the XFC provider of the embedded content via Consumer.on('eventName', () => {});

See example: https://github.com/cerner/terra-framework/pull/545/files#diff-b785ae9e7da3f4e819d40fcadaf318a3R10

Reply:
We already pass the frame back in the onMount callback: https://github.com/cerner/terra-framework/blob/0fd733b9e40212a806fb6bb9cb9557887ea94948/packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx#L46

I guess the expectation today would be that consumers would hold on to that frame locally and trigger events as needed. If that's the case, do we need to send the frame with the generic event callbacks?

Reply:
@tbiethman I just realized this myself about 30 minutes ago working on other stuff. I didn't read the docs closely. Removing this change.

Reply:
Updated: https://github.com/cerner/terra-framework/pull/545/commits/b0627d752ac00b494b1ffe009102e267b4a1ad48 && https://github.com/cerner/terra-framework/pull/545/commits/ee00fcc669d0c02762be3c05c4b884c22f4411d4